conformance/lib/src/nativeMain/kotlin/pbandk/conformance/Platform.kt

```diff
@@ -92,6 +101,13 @@ actual object Platform {
         }
     }
 
+    actual fun <T : Message> stdoutWriteLengthDelimitedMessage(message: T) {
+        message.encodeToByteArray().also { bytes ->
+            stdoutWriteIntLE(bytes.size)
+            stdoutWriteFull(bytes)
```

Comment:
So previously the `Platform` interface had separate read length, and read n bytes methods, but those were always used to just compose a message?  So now you merged them to just have the interface declare the read/write message?  Is that basically the summary of these `Platform` changes?

Reply:
Yep exactly